src/components/PrintForm/index.tsx

```diff
@@ -53,23 +69,84 @@ export interface PrintFormProps extends Omit<FormProps, 'form'> {
   layerBlackList?: string[];
   outputFormats?: string[];
   layouts?: Layout[];
-  printManager: MapFishPrintV3Manager;
+  active: boolean;
 }
 
 export const PrintForm: React.FC<PrintFormProps> = ({
   layerBlackList = [],
   map,
-  printManager,
+  active,
   ...restProps
 }): JSX.Element => {
   const [form] = Form.useForm();
   const {
     t
   } = useTranslation();
+  const client = useSHOGunAPIClient();
 
+  const [printManager, setPrintManager] = useState<MapFishPrintV3Manager | null>(null);
   const [loading, setLoading] = useState<boolean>(false);
   const [errorMsg, setErrorMsg] = useState<string | null>(null);
 
+  const layerFilter = useCallback((l: OlLayer<OlSource>) => {
+    if (!map) {
+      return;
+    }
+
+    const layerName = l.get('name');
+
+    return layerName && !layerBlackList.includes(layerName) &&
+      l.getVisible() && !(l instanceof OlLayerGroup) &&
+      MapUtil.layerInResolutionRange(l, map);
+  }, [map, layerBlackList]);
+
+  const initializeMapProvider = useCallback(async () => {
+    // @ts-ignore
+    const pManager: MapFishPrintV3Manager = new MapFishPrintV3Manager({
+      url: ClientConfiguration.print?.url || '/print',
+      map,
+      timeout: 60000,
+      layerFilter,
+      headers: {
+        ...getBearerTokenHeader(client?.getKeycloak())
+      },
+      transformOpts: {
+        rotate: false
+      }
+    });
+
+    pManager.serializers = [
+      new MapFishPrintV3GeoJsonSerializer(),
+      new MapFishPrintV3OSMSerializer(),
+      new MapFishPrintV3WMTSSerializer(),
+      new SHOGunMapFishPrintV3WMSSerializer(),
+      new SHOGunMapFishPrintV3TiledWMSSerializer()
+    ];
+
+    await pManager.init()
+      .then(() => {
```

Comment:
Since we make use of `await` `then()` is not needed.

Reply:
Nice (copypasted) catch! Fixed with [e20f4b5](https://github.com/terrestris/shogun-gis-client/pull/528/commits/e20f4b5c1e0fa094c9df89a14741fd9a97fb63cc)